app/gateway-oss/2.3.x/upgrading.md

```diff
@@ -77,58 +78,55 @@ repository contains [openresty-build-tools](https://github.com/Kong/kong-build-t
 which allows you to build OpenResty with the necessary patches
 and modules easily.
 
-For Go support, you also need to build both your plugins and
-the [Kong go-pluginserver](https://github.com/kong/go-pluginserver).
-The documentation includes detailed [instructions on how to build
-the plugin server and your plugins](https://docs.konghq.com/2.2.x/go/).
+There is a new way to deploy Go using Plugin Servers. You can read more about it the [Go section of the docs](https://docs.konghq.com/2.3.x/go).
 
-#### 2. Template Changes
+### Template changes
 
 There are **Changes in the Nginx configuration file**, between kong 2.0.0,
-2.1.0 and 2.2.0.
+2.1.0, 2.2.0 and 2.3.0.
```

Comment:
Do we know for sure if there are nginx config fie changes for 2.3.? This should be verified by eng.

Reply:
I confirm that there have been changes in the Nginx configuration file in 2.1.0, 2.2.0 and 2.3.0